plugins/mentions.md

```diff
@@ -93,6 +91,59 @@ tinymce.init({
 });
 ```
 
+### `mentions_item_type`
+
+{{site.requires_5_6v}}
+
+This option allows you to specify what kind of user interface item to use when displaying the list of users. The `name` item will just display the user's name, while the `profile` item can also display an optional image and description. Depending on what this option is set to, additional properties may be required for the user object provided via [mentions_fetch](#mentions_fetch). Refer to the [User properties](#userproperties) table for more details.
+
+**Type:** `String`
+
+**Default Value:** `name`
+
+**Possible Values:** `name`, `profile`
+
+#### Example: Using `mentions_item_type`
+
+```js
+tinymce.init({
+  selector: 'textarea',
+  plugins: 'mentions',
+  mentions_item_type: 'name'
+});
+```
+
+##### User properties
+
+The user properties table describes which properties you may need to define for the user objects provided via the [mentions_fetch](#mentions_fetch) callback. Which properties are required, optional and not available depends on the [mentions_item_type](#mentions_item_type) setting, which may be of type `name` or `profile`, and the [mentions_select](#mentions_select) setting, which may be of type `profile`.
+
+| Name | Value | `name` | `profile` | Description |
+| ---- | ----- | ----------- | ----------- | ----------- |
+| id | string | required | required | Used to identify the user mention in different callbacks |
+| name | string | required | required |  Name to display and highlight search matches on |
```

Comment:
`search matches` reads a little bit odd. I don't think we need both words, so maybe just make this something like this?
```suggestion
| name | string | required | required |  Name to display and highlight matches |
```